### PR TITLE
Upgrade stack

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Upgrades most of the deps including: react, chakra, next, and node (for Netlify).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,19 +25,20 @@
...
-    "react-dom": "17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-dropzone": "^12.0.2",
     "react-hook-form": "^7.22.1",
     "react-intersection-observer": "^8.33.1",
     "react-markdown": "^8.0.0",
-    "redaxios": "^0.4.1"
+    "redaxios": "^0.4.1",
+    "sharp": "^0.33.2"
```
Comment (pettinarip) [inline package.json:41]: https://nextjs.org/docs/messages/sharp-missing-in-production

**Review by wackerow — DISMISSED** [review]
@pettinarip Nice job, lgtm! Left a comment that does not need to be addressed here. Overall functioning as expected when testing out the site.

File: src/components/Nav.tsx [hunk 2]
```diff
@@ -105,35 +105,34 @@ export const Nav: FC = () => {
...
-                  <Link
-                    href={nav.href}
-                    target={nav.href === ESP_BLOG_URL ? '_blank' : '_self'}
-                    rel='noopener noreferrer'
-                    _hover={{ textDecoration: 'none' }}
+                <Link
+                  as={NextLink}
+                  href={nav.href}
+                  target={nav.href === ESP_BLOG_URL ? '_blank' : '_self'}
+                  rel='noopener noreferrer'
```
[the remark this answers is not in the document]
Comment (pettinarip) [reply src/components/Nav.tsx:112]: Good observation, true. I can add that. Did a quick search and it seems that it is the only link that was using the `target` prop 👍🏼

**Comment by pettinarip** [comment]
FYI, marking this as **blocked** for now since the build is failing. I've retried the build cleaning the cache but it failed again. Investigating...

**Comment by pettinarip** [comment]
**Blocking** this once again, found a bug.

The functions are now deployed but when I tried to submit an application it fails with the following error:
```
Error: Cannot find module 'next/dist/server/future/route-modules/pages/vendored/contexts/amp-context'
```

Investigating...

**Pull request merged**